src/test/java/org/assertj/core/api/object/ObjectAssert_extracting_Test.java

```diff
@@ -70,10 +70,10 @@ public void should_use_property_field_names_as_description_when_extracting_tuple
   public void should_keep_existing_description_if_set_when_extracting_tuples_list() {
     Employee luke = new Employee(2L, new Name("Luke", "Skywalker"), 26);
 
-    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThat(luke).as("check luke first name")
-                                                                                     .extracting("name.first")
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThat(luke).as("check luke first and last name")
+                                                                                     .extracting("name.first", "name.last")
                                                                                      .isEmpty())
-                                                   .withMessageContaining("[check luke first name]");
+                                                   .withMessageContaining("[check luke first and last name]");
   }
 
```

Comment:
Let's add test specific to the new `extracting` method (as you did in `MapAssert_extracting_Test`)

Reply:
Side question: should the test `should_allow_assertions_on_array_of_properties_extracted_from_given_object_with_lambdas` stay in `ObjectAssert_extracting_with_function_Test` instead of `ObjectAssert_extracting_Test`?

Reply:
I moved the test to the other class and this triggered an additional change regarding wildcards in `extracting(Function[])` methods, please double check it!